plugin/trino-hive/src/main/java/io/trino/plugin/hive/BackgroundHiveSplitLoader.java

```diff
@@ -472,6 +473,8 @@ private void invokeNoMoreSplitsIfNecessary()
 
             JobConf jobConf = toJobConf(configuration);
             FileInputFormat.setInputPaths(jobConf, path);
+            // SerDes parameters and Table parameters passing into input format
```

Comment:
What is broken without it? Could you add a test?

Reply:
Hi @sopel39 , 

- It is not breaking anything. For custom hive input format unable to get SerDes parameters and Table parameters as part split. In hive query engine `jobconf` for get split function provides the SerDes parameters and Table parameters. So same way, presto also provide this parameters. This bring new capability like schema validation at input format using `serialization.ddl` and do some customisation with help of table properties. 

- I don't thing we need test cases for this changes. Let me know.

Reply:
Is there a serde that breaks without this change?

Reply:
Nope. I am trying to enable `deltalake` trino connector using this feature. [https://github.com/delta-io/connectors](url)

- `InputFormat#getSplits` method needs `serialization.ddl` properties or `columns` and `columnTypes` from table schema for validating schema with delta lake table schema
- Table properties are required for time travel feature at `InputFormat#getSplits` method

Trying to have hive schema information at `InputFormat#getSplits` level for new file format like delta lake support.

Reply:
We are passing serde parameters into `InputFormat#gerRecordReader` at https://github.com/trinodb/trino/blob/master/plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveUtil.java#L227 . Similar way, new file format like delta lake and custom hive format get benefit of this parameters at InputFormat#getSplits for new file format support.

Reply:
Are `SerDes parameters and Table parameters passing into input format` passed by Hive when running queries?

Reply:
@sopel39 yes, that's correct. We need same behaviour in trino. I have recently updated prestodb https://github.com/prestodb/presto/pull/15852